### ISSUE TITLE
Notification server not running with the `notfiy_below` of `battery` widget.

### ISSUE THREAD

**Issue by a2n-s** [qtile/qtile#3348]
I did not see a really similar issue, so I give it a go! :yum: 

I set the [`notfiy_below`](https://github.com/a2n-s/dotfiles/blob/b92e85def5893a9f10698864e08625137c95b729/.config/qtile/widgets.py#L954) argument for the `libqtile.widget.Battery` widget and I get the following log:

```
2022-02-17 21:06:22,661 WARNING libqtile utils.py:_notify():L282 Unable to send notification. Is a notification server running?
```

I installed `dbus` with `pip install dbus-next` as said in the doc, but I do not know what to do at all...

**Any ideas? Or if it's a bug?**

**Comment by a2n-s** [comment]
I have to say I never use notifications, this is the first time I try them. So I do not even now if any notification server is running at all. I assume though as some other commands are able to send notifications without any issue.

Running `notify-send body msg` does nothing at all and does not send an error code either.

I think I installed `dbus-next` on the first day of `qtile` as I was going through the install process on the documentation and did not want to have "some features not working". So yes it should be installed :thinking:

**Comment by m-col** [comment]
OK, no worries. A notification server is, as the name suggests, a server that handles notifications. It's what responds when something like `notify-send` sends a notification. The `Battery` widget similarly sends a notification, but a notification server is needed to take action. We have the `Notify` widget which does this very thing - it listens for notifications via `dbus` and any that are made are displayed in the widget. There are also more fancy servers like `dunst` which will pop up notification windows when they are made. You'll need something like that.

**Comment by a2n-s** [comment]
Oh wow, I did not get that the `libqtile.widget.Notify` widget was that important.
Now I added it in the bar and I think I saw a popup warning in the bar saying the battery was low!

Thanks a lot!

I would really like to have a popup in the middle of the screen, e.g. when watching something fullscreen on a single monitor, I want to be interrupted so that I can charge my laptop. It really does not behave well on low battery...

So I'll try to look at `dunst` then :ok_hand:
